tests/scanners/test.binary.js

```diff
@@ -0,0 +1,35 @@
+import BinaryScanner from 'scanners/binary';
+import {FLAGGED_FILE_MAGIC_NUMBERS} from 'const';
```

Comment:
This is a major nitpick but we usually put spaces around grouped imports like this:

```js
import { FLAGGED_FILE_MAGIC_NUMBERS } from 'const';
```